resource-monitoring/README.md

```diff
@@ -0,0 +1,40 @@
+## OBJECTIVE
+How about monitoring all you system resources through real-time graphs?.Do you know this could be done just by a small effort - lets start monitoring! :open_mouth:
+
```

Comment:
What about this?

You are running too many applications and your system start running slow. You want to know what is running on your system. We have a tool just for this.